.circleci/config.yml

```diff
@@ -258,3 +277,7 @@ workflows:
   idea:
     jobs:
       - run-inspections
+
+  git-validation:
```

Comment:
created a new workflow to accommodate https://github.com/checkstyle/checkstyle/issues/12319